sale_order_mass_action/wizards/sale_order_mass_action_wizard.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderMassActionWizard(models.TransientModel):
+
+    _name = "sale.order.mass.action.wizard"
+    _description = "Sale Order Mass Action"
+
+    confirm = fields.Boolean(
+        help="Check this box if you want to confirm all the selected quotations."
+    )
+
+    def _get_sale_order_confirm_domain(self):
+        return [
+            ("id", "in", self.env.context.get("active_ids")),
+            ("state", "in", ("draft", "sent")),
+        ]
+
+    @api.model
+    def _notify_success(self, sale_orders):
+        order_names = "\n".join(sale_orders.mapped("name"))
+        message = "The following orders has been updated : %s" % order_names
```

Comment:
@jappi00 Could you improve this in order to get translation ?

```suggestion
        message = _("The following orders has been updated : %(order_names)s", order_names=order_names)
```

Reply:
> @jappi00 Could you improve this in order to get translation ?

In a separate commit, thanks